test/index.js

```diff
@@ -528,6 +529,19 @@ describe('def', function() {
     eq(id([{x: 0, y: 0}, {x: 1, y: 1}]), [{x: 0, y: 0}, {x: 1, y: 1}]);
   });
 
+  it('supports "ValidDate" types', function () {
+    var sinceEpoch = def('sinceEpoch', {}, [$.ValidDate, $.Number], function(date) {
+      return date.getTime() / 1000 });
+
+    throws(function() { sinceEpoch(new Date('foo')); },
+      errorEq(TypeError,
+        '‘sinceEpoch’ expected a value of type ValidDate ' +
+        'as its first argument; received new Date(NaN)'));
```

Comment:
:zap:  What editor are you using? Do you have a way of exporting your formatting rules? I'm always messing up when it comes to this sort of thing.